chain/cosmos/08-wasm-types/wasm.pb.go

```diff
@@ -7,7 +7,7 @@ import (
 	fmt "fmt"
 	_ "github.com/cosmos/gogoproto/gogoproto"
 	proto "github.com/cosmos/gogoproto/proto"
-	types "github.com/cosmos/ibc-go/v7/modules/core/02-client/types"
+	types "github.com/cosmos/ibc-go/v8/modules/core/02-client/types"
```

Comment:
not sure if this needs to be regenerated. I just did a find/replace

Reply:
I don't think that it **needs** to, but I ususally do when I make similar PRs

Reply:
cc @misko9 is this fine with not being regenerated for now? It will need a regen eventually in the future anyway to match the latest [feat/wasm-clients](https://github.com/cosmos/ibc-go/tree/feat/wasm-clients) branch anyway